target/linux/mvebu/image/cortex-a53.mk

```diff
@@ -2,42 +2,64 @@ ifeq ($(SUBTARGET),cortexa53)
 
 define Device/globalscale_espressobin
   $(call Device/Default-arm64)
-  DEVICE_TITLE := ESPRESSObin (Marvell Armada 3700 Community Board)
+  DEVICE_VENDOR := Marvell
+  DEVICE_MODEL := ESPRESSObin
+  DEVICE_VARIANT := Non-eMMC
+  DEVICE_ALT0_VENDOR := Marvell
+  DEVICE_ALT0_MODEL := Armada 3700 Community Board
+  DEVICE_ALT0_VARIANT := Non-eMMC
   DEVICE_DTS := armada-3720-espressobin
 endef
 TARGET_DEVICES += globalscale_espressobin
 
 define Device/globalscale_espressobin-emmc
   $(call Device/Default-arm64)
-  DEVICE_TITLE := ESPRESSObin eMMC (Marvell Armada 3700 Community Board)
+  DEVICE_VENDOR := Marvell
+  DEVICE_MODEL := ESPRESSObin
+  DEVICE_VARIANT := eMMC
+  DEVICE_ALT0_VENDOR := Marvell
+  DEVICE_ALT0_MODEL := Armada 3700 Community Board
+  DEVICE_ALT0_VARIANT := eMMC
   DEVICE_DTS := armada-3720-espressobin-emmc
 endef
 TARGET_DEVICES += globalscale_espressobin-emmc
 
 define Device/globalscale_espressobin-v7
   $(call Device/Default-arm64)
-  DEVICE_TITLE := ESPRESSObin V7 (Marvell Armada 3700 Community Board)
+  DEVICE_VENDOR := Marvell
+  DEVICE_MODEL := ESPRESSObin
+  DEVICE_VARIANT := V7 Non-eMMC
+  DEVICE_ALT0_VENDOR := Marvell
+  DEVICE_ALT0_MODEL := Armada 3700 Community Board
+  DEVICE_ALT0_VARIANT := V7
```

Comment:
I'm fine with the "V7 Non-eMMC", but you also have to add it for ALT0 then.

Reply:
ack